src/activities/instanceCollections/CollectedObject.js

```diff
@@ -1,13 +1,13 @@
 import { Entity } from '../../es6/Entity.js';
 
 export class CollectedObjectEntity extends Entity {
-	getSortOrder() {
+	sortOrder() {
```

Comment:
Hmm I personally like having the action verb at the start... 👀

Reply:
Thats what I usually prefer for methods, but I was looking at some of the other files and they do it like this.

For example: https://search.d2l.dev/xref/BrightspaceHypermediaComponents/siren-sdk/src/activities/ActivityEvaluationStatusEntity.js?r=6d5e120f#28

What are your thoughts? 🤔

Reply:
Ah you know what that is true. As long as we are dealing with properties, I think this is fine to be consistent 👍